tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -963,5 +963,17 @@
       "4.20.*",
       "4.*.*"
     ]
+  },
+  {
+    "IntegrationName": "OpenFeature",
+    "SampleProjectName": "Samples.OpenFeature",
+    "NugetPackageSearchName": "OpenFeature",
+    "MinVersion": "1.0.0",
+    "MaxVersionExclusive": "3.0.0",
+    "SpecificVersions": [
+      "1.0.*",
```

Comment:
Are we actually going to support 1.x of OpenFeature, or are we going to require 2.x?

Reply:
The most version we can support with the same code, the better